demo/billing/src/mz.rs

```diff
@@ -195,7 +195,7 @@ fn billing_agg_view(unit: &str) -> String {
     format!("CREATE MATERIALIZED VIEW billing_agg_by_{unit} AS
 SELECT date_trunc('{unit}', interval_start::timestamp) AS {unit}, client_id, meter, cpu_num, memory_gb, disk_gb, sum(value)
 FROM billing_records
-GROUP BY {unit}, client_id, meter, cpu_num, memory_gb, disk_gb",
+GROUP BY \"{unit}\", client_id, meter, cpu_num, memory_gb, disk_gb",
```

Comment:
As alluded to previously the fact that you had to introduce quoting here is worrisome to me! There could very well be user views out there that use months as column names and somehow PostgreSQL is able to support that.

Restricting date/time fields for use in aliases is also backwards incompatible of course, but that one is unavoidable, it seems! We should get a release note mentioning that though.

Reply:
This should be fixed up now. Details are in the amended commit message.